modeltranslation/admin.py

```diff
@@ -167,7 +167,7 @@ def replace_orig_field(self, option: Iterable[str]) -> _ListOrTuple[str]:
             for opt in option:
                 if opt in self.trans_opts.fields:
                     index = option_new.index(opt)
-                    option_new[index : index + 1] = get_translation_fields(opt)
+                    option_new[index : index + 1] = get_translation_fields(opt)  # type: ignore[arg-type]
```

Comment:
`pyright` is smart enough to narrow `opt` to `str` thanks to the `if opt in self.trans_opts.fields` check, but not `mypy`